### DISCUSSION TITLE
Using with VMware Horizon

### DISCUSSION THREAD

**Discussion by TwoDogs2** [umlaeute/v4l2loopback#588] [Q&A] [answered]
I encountered some issues during the installation process, and the patch did not echo and stuck in this interface Is there any good solution? thanks
![v4](https://github.com/umlaeute/v4l2loopback/assets/121689365/3c2376c6-4c30-4d33-acf1-8003bcb003bd)

**Comment by umlaeute** [comment, marked as answer]
i have no idea what you are trying to do.
what i gather is, that you want to apply some random patch following some unknown tutorial to get an outdated version of `v4l2loopback` running with some commercial software.

since `v4l2loopback` is run by unpaid volunteers, i would kindly ask you to to follow these general rules:
- please do not post screenshots of your console. instead post the actual text!
- use the latest and greatest `v4l2loopback` unless you have reasons to do otherwise. if you have such reasons, tell us!
- provide information about what you actually trying to do. if you are following a tutorial or some 3rd party instructions, tell us where these instructions can be found!
- you probably should first ask the person who wrote these instructions
- if this doesn't help and your problem is with a **commercial** product, try contacting them (they sell things, they earn money, they are supposed to provide support)

in your specific case this boils down to:
- please post text.
- `v4l2loopback-0.12.5` is not the latest and greatest (though this is actually unrelated to your problem)
- it is quite obvious that you *are* following some instructions. i don't think that these instructions are somewhere *here*; so we do not know what you are doing. ask whoever wrote these instructions.
- *VMWare Horizon* is a commercial product. contact their support

your specific problem is how you invoke `patch`.
you probably should read the documentation:

```sh
man 1 patch
```

(I'm sorry I do not really feel like expanding on this)
